src/tbb/CMakeLists.txt

```diff
@@ -79,14 +79,16 @@ target_compile_options(tbb
     ${TBB_COMMON_COMPILE_FLAGS}
 )
 
+if (FALSE)
 # Avoid use of target_link_libraries here as it changes /DEF option to \DEF on Windows.
 set_target_properties(tbb PROPERTIES
     DEFINE_SYMBOL ""
     VERSION ${TBB_BINARY_VERSION}.${TBB_BINARY_MINOR_VERSION}
     SOVERSION ${TBB_BINARY_VERSION}
```

Comment:
These properties are required for each library to generate the correct name for the library. It causes backward compatibility tests, arena constraint tests and test_malloc_lib_unload to fail.
Beside above issues, testing seems Ok on Windows, Linux and macOS.

Reply:
Thanks for the review, I'll fix it.